README.md

```diff
@@ -76,6 +76,40 @@ Documentation about the pipeline, found in the [`docs/`](docs) directory:
 3. [Running the pipeline](docs/usage.md)
     * [Running on Sumner](docs/run_on_sumner.md)
     * [Running on CloudOS](docs/run_on_cloudos.md)
+    * [Running locally](docs/run_locally.md)
 
 ## Pipeline DAG
-<img src="https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/blob/c23ee9552eb033dec087fb3b6fb01fe26716ce29/DAG.png" alt="splicing_pip_dag" align = "left" width="600"/>
+<img src="https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/blob/c23ee9552eb033dec087fb3b6fb01fe26716ce29/DAG.png" alt="splicing_pip_dag" align = "center" width="600"/>
+
+
+## Changelog
```

Comment:
changelog can be a separate file in the root of the repo named `changelog.md`.

Add also the field:

```
PR: https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/pull/245
```
to track the PR that introduced the changes easily.

Reply:
done